packages/domain/src/use-cases/profile/ActiveProfileLoader.ts

```diff
@@ -12,14 +12,26 @@ export class ActiveProfileLoader {
     private readonly activeProfilePresenter: IActiveProfilePresenter,
   ) {}
 
-  async loadActiveProfileByOwnerAddress(walletAddress: string) {
+  async loadActiveProfileByOwnerAddress(walletAddress: string, profileId?: string) {
     const activeProfile = await this.activeProfileGateway.getActiveProfile();
 
     if (activeProfile) {
       await this.activeProfilePresenter.presentActiveProfile(activeProfile);
       return;
     }
-    const [firstProfile] = await this.profileGateway.getAllProfilesByOwnerAddress(walletAddress);
+    const profiles = await this.profileGateway.getAllProfilesByOwnerAddress(walletAddress);
```

Comment:
wondering if we can use `handle` over `profileId` which seems to be closer to how protocol works.

We already have a method `loadActiveProfileByHandle` that we can call in that case and keep this one to load first active profile by the wallet address. 

nit: maybe it's also worth checking in `loadActiveProfileByHandle` if the `handle` belongs to the active wallet and throw some invariant error.

Reply:
We could go also one step further and remove `loadActiveProfileByOwnerAddress` and lift the logic to select the first profile handle to the caller side and call `loadActiveProfileByHandle` directly.